### PR TITLE
CSS and HBS for ESMF website in regards to landing page redesign

### PR DESCRIPTION
**Pull request by anastascho**
This is the current state of all css and hbs changes intended for the ESMF landing page redesign.
Please review them, but keep in mind that this is still in progress.

### CODE REVIEW

**Comment by anastascho** [comment]
Here you can find a snapshot of the current state of the landing page:

http://fe0vm2112.de.bosch.com/doku/bosch-semantic-stack/ESMF/site/site/esmf-documentation/index.html

File: src/partials/sections-nav-mobile.hbs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="sections-navbar-mobile">
+            <ul class="sections-nav-list-mobile">
+          <li><a href="https://projects.eclipse.org/proposals/eclipse-semantic-modeling-framework-esmf" target="_blank">About ESMF</a></li>
```
Comment (aniluede) [inline src/partials/sections-nav-mobile.hbs]:
```suggestion
          <li><a href="https://projects.eclipse.org/projects/dt.esmf" target="_blank">About ESMF</a></li>
```

File: src/partials/sections-header-content.hbs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      </a>
+      {{!-- <button class="sections-navbar-burger" data-target="sections-topbar-nav"> --}}
+        <span></span>
+        <span></span>
+        <span></span>
+      </button>
+    </div>
+    <div id="sections-topbar-nav" class="sections-navbar-menu">
+        <ul class="sections-nav-list">
+          <li><a href="https://projects.eclipse.org/proposals/eclipse-semantic-modeling-framework-esmf" target="_blank">About ESMF</a></li>
```
Comment (aniluede) [inline src/partials/sections-header-content.hbs]:
```suggestion
          <li><a href="https://projects.eclipse.org/projects/dt.esmf" target="_blank">About ESMF</a></li>
```

**Review by aniluede — CHANGES_REQUESTED** [review]
two links to be updated

**Review by aniluede — APPROVED** [review]
LGTM

**Pull request merged**